simple_config

```diff
@@ -6,6 +6,7 @@ num_threads 5;
 
 path /static StaticRequestHandler {
   root /test;
+  cache_max_age 5;
```

Comment:
Can we add a comment here to specify the units? 

For example: `cache_max_age 5; # seconds, no commas or periods`